hhs_oauth_server/settings/base.py

```diff
@@ -79,6 +79,7 @@
     'apps.accounts',
     'apps.capabilities',
     'apps.wellknown',
+    'apps.testclient',
```

Comment:
I sent an email with more detail, but this is not something that I am comfortable with.